tests/unit/core/test_searchUtils.js

```diff
@@ -41,4 +44,46 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('convertOSToFilterValue', () => {
+    function getOSNameFromUserAgent(userAgent) {
+      const { store } = dispatchClientMetadata({ userAgent });
+      return store.getState().api.userAgentInfo.os.name;
+    }
+
+    it('converts Windows to filter', () => {
+      const osName = getOSNameFromUserAgent(userAgents.firefox[1]);
+      const filterName = convertOSToFilterValue(osName);
```

Comment:
No biggie but `filterName` in all of these look like they're left over from before the name change

Reply:
Hah, good catch.